builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -293,6 +319,20 @@ func (b *backend) pathUpdateIssuer(ctx context.Context, req *logical.Request, da
 		return nil, err
 	}
 
+	// AIA access changes
+	issuerCertificates := data.Get("issuing_certificates").([]string)
+	if badURL := validateURLs(issuerCertificates); badURL != "" {
+		return logical.ErrorResponse(fmt.Sprintf("invalid URL found in issuing certificates: %s", badURL)), nil
```

Comment:
The error message doesn't read well to someone without knowledge of the code - consider either changing `issuing certificates` to `issuing_certificates` so it describes the parameter by literal name, or to `Authority Information Access` so that it describes the parameter by concept as referred to in e.g. RFCs.

Any change should also be applied to the copy of this code in the `PATCH` method handler.

Reply:
Yeah, most of these error messages were copied from existing places; good idea to update them.